biocypher_metta/adapters/roadmap_h3_marks_adapter.py

```diff
@@ -52,14 +52,14 @@ def get_edges(self):
                         _id = row[0]
                         chr = self.dbsnp_rsid_map[_id]["chr"]
                         pos = self.dbsnp_rsid_map[_id]["pos"]
-                        cell_id = row[COL_DICT['cell']].replace('"', '').replace("'", '')
```

Comment:
Any reason for this change?

Reply:
It was because of a bug that was causing the tests to fail in the roadmap adapter. the cell_to_ontology_id_map dictionary uses only the cell ID prefix as key not the full cell description